### PR TITLE
Add Oneclick Deferred Capture

### PR DESCRIPTION
**Pull request by afiebig**
- Normalize Classes name to Mall<Something> in order to be consistent with the other products
- Add Oneclick Deferred
- Also add obsolete mark to old clases.

### CODE REVIEW

**Comment by gonzunigad** [comment]
I'm not sure but i think you swaped the files MallCaptureException.cs  and MallTransactionCreateException.cs.

File: Transbank/WebpayRest/Oneclick/Requests/MallFinishRequest.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace Transbank.Webpay.Oneclick.Requests
+{
+    internal class MallFinishRequest : BaseRequest
+    {
+        internal MallFinishRequest(string token)
+            : base($"/rswebpaytransaction/api/oneclick/v1.0/inscriptions/{token}",
+                  HttpMethod.Put){}
+    }
+}
```
Comment (gonzunigad) [inline Transbank/WebpayRest/Oneclick/Requests/MallFinishRequest.cs:12]: What is the difference between this `MallFinishRequest` class and `FinishRequest` if the only Oneclick available is Mall?

Comment (afiebig) [reply Transbank/WebpayRest/Oneclick/Requests/MallFinishRequest.cs:12]: This was done to retain consistency with the others products. We discussed it on Slack.

**Review by gonzunigad — DISMISSED** [review]
LGTM 👍

**Review by gonzunigad — APPROVED** [review]

**Pull request merged**